assets/xml/objects/object_b_heart.xml

```diff
@@ -0,0 +1,6 @@
+<Root>
+    <File Name="object_b_heart" Segment="6">
+        <DList Name="gBHeartDL1" Offset="0x240"/>
```

Comment:
The names should try to express what the object actually is, rather than using Nintendo's names, for here you would want gHeartPieceCollectibleDL1 or something similar I think.